ext/systemd/puppetmaster.service

```diff
@@ -5,7 +5,8 @@ After=basic.target network.target
 
 [Service]
 EnvironmentFile=-/etc/sysconfig/puppetmaster
-ExecStart=/usr/bin/puppet master $PUPPETMASTER_EXTRA_OPTS --no-daemonize
+EnvironmentFile=-/etc/default/puppetmaster
+ExecStart=/opt/puppetlabs/puppet/bin/puppet master $PUPPETMASTER_EXTRA_OPTS --no-daemonize
```

Comment:
Seems like we should just kill the puppetmaster service file all together. It's not really a thing anymore.